app/models/piece.rb

```diff
@@ -1,4 +1,38 @@
+# rubocop:disable all
 class Piece < ActiveRecord::Base
   belongs_to :game
   belongs_to :player
+
+  def obstructed?(x_destination, y_destination)
+    pieces = []
+    x = x_position
+    y = y_position
+
+    if moving_horizontally?(x_destination, y_destination)
```

Comment:
Inside of the `moving_horizontally?` method, only the `y_destination` argument is used. Just seems cleaner to change `moving_horizontally?` to accept only 1 argument and change the `if moving_horizontally?` method call to pass only 1 argument, but I wasn't sure if I was missing something. Same would go for the `moving_vertically?` method...